src/main/frontend/fs/capacitor_fs.cljs

```diff
@@ -148,7 +185,15 @@
          (p/let [result (.writeFile Filesystem (clj->js {:path path
                                                          :data content
                                                          :encoding (.-UTF8 Encoding)
-                                                         :recursive true}))]
+                                                         :recursive true}))
+                 mtime (-> (js->clj stat :keywordize-keys true)
+                           :mtime)]
+           (when-not contents-matched?
+             (when (and (string? disk-content)
+                        (string? content)
+                        (string-some-changed? disk-content content))
```

Comment:
Curious why this is needed.

Reply:
If the change exceeds 10 chars, backup, then not.

Reply:
It's still possible to lose some small changes, right? Because we don't have page history for all the platforms yet.

Reply:
Correct. I was worried that if we backup all changes, some important backups might be deleted since we only backup three version.